packages/backup-format/src/container/index.ts

```diff
@@ -1,8 +1,8 @@
 import { concatArrayBuffer, unreachable } from '@dimensiondev/kit'
 import { BackupErrors } from '../BackupErrors'
 
-const MAGIC_HEADER_Version0 = new Uint8Array([...'MASK-BACKUP-V000'].map((x) => x.charCodeAt(0)))
-const CHECKSUM_LENGTH = 16
+const MAGIC_HEADER_Version0 = new TextEncoder().encode('MASK-BACKUP-V000')
```

Comment:
I prefer the `Uint8Array` way. It's JS-only API but `TextEncoder` relies on Web API.